src/main/app/form/Form.ts

```diff
@@ -47,14 +49,13 @@ export interface FormInput {
   selected?: boolean;
   value?: string | number;
   validator?: ValidationCheck;
+  warning?: Warning;
 }
 
 export interface CsrfField {
   _csrf: string;
 }
 
-export type FormBody<T extends FormContent> = Record<keyof T['fields'], string> & CsrfField;
```

Comment:
and I thought I was so clever

Reply:
I couldn't find a way to reuse these types with dynamic steps 😞